akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardingQueries.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+
+import akka.annotation.InternalApi
+
+/** INTERNAL API */
+@InternalApi
+private[sharding] object ShardingQueries {
+
+  private def func[T] = (t: T) => t
+
+  def separateE[A, B](ps: Seq[Either[A, B]]): (Seq[A], Seq[B]) =
+    separate(ps)(func)
```

Comment:
I think this could be replaced by `seq.partitionMap(identity)`

Reply:
Didn't know about `partitionMap`, sounds useful

Reply:
Nice, thanks. Updated.

Reply:
Added in 2.13